.github/workflows/build-alignscore.yaml

```diff
@@ -0,0 +1,47 @@
+name: Deploy on push to main
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "**"
+    paths:
+      - optional_components/alignScore/**
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    environment: dev
+    steps:
+      # Add a setup buildx step
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ vars.AWS_ACCOUNT_ID }}:role/github_actions_dev
+          role-session-name: github-actions
+          aws-region: us-east-1 # ${{ vars.AWS_REGION }}
+      - name: Login to Amazon ECR
+        id: login-ecr-public
+        uses: aws-actions/amazon-ecr-login@v2
+        with:
+          registry-type: public
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        env:
+          REGISTRY: ${{ steps.login-ecr-public.outputs.registry }}
+          REGISTRY_ALIAS: j3r7b4k0
+          REPOSITORY: alignscore-base
+          IMAGE_TAG: latest
```

Comment:
I don't really like that it updates the 'latest' image even on a branch that is not `main`. 
After we have alignScore down, I'll fix it to
- recreate `latest` on merge to `main`
- set dependency between jobs. So build/push image before running tests